js/widgets/widget.backcompat.js

```diff
@@ -116,6 +142,12 @@ if ( $.mobileBackcompat !== false ) {
 			}
 
 			this._superApply( arguments );
+		},
+
+		destroy: function() {
+			if ( this.options.wrapperClass && !this.options.enhanced ) {
+				this.widget().removeClass( this.options.wrapperClass );
+			}
 		}
```

Comment:
This whole method can be dropped once you fix the addition of the class above. Also you should never use the "destroy" method this is part of the base widget always use `_destroy` which is called from `destroy` internally in the base widget. The above code will break the `destroy` method in every widget that uses this.